scripts/power_test.py

```diff
@@ -1,54 +1,143 @@
-#!/usr/bin/env python
-import pyudev
-from power import Power
+#!/usr/bin/env python3
+
+"""
+Power board test script.
+
+This will:
+
+ - hopefully not crash
+"""
+
 import time
+from typing import Any, Iterable, NamedTuple, Optional
+
+import usb.core
+import usb.util
+
+class USBDevice(NamedTuple):
```

Comment:
Would be great if we could make this compatible with Python 3.5 so those of us using Debian stable (or a derivative) can use these scripts.

``` python
USBDevice = NamedTuple('USBDevice', (
    ('serial', str),
    ('node', usb.core.Device),
    ('busnum', int),
    ('devnum', int),
))
```